build-settings/rollup.config.js

```diff
@@ -23,7 +23,63 @@ const createConfig = (pkgName) => ({
             packagePath: `packages/${pkgName}/package.json`,
         }),
         terser(),
+
+        // This generates the flow import file.
+        // The copyOnce ensures that we just get one copy of the requested
+        // target file, so it's ok that we duplicate this step in our two
+        // outputs.
+        copy({
+            copyOnce: true,
+            verbose: true,
+            targets: [
+                {
+                    // src path is relative to the package root unless started
+                    // with ./
+                    src: "build-settings/index.flow.js.template",
+                    // dest path is relative to src path.
+                    dest: `packages/${pkgName}/dist`,
+                    rename: "index.flow.js",
+                },
+            ],
+        }),
     ],
 });
 
-export default fs.readdirSync("packages").map(createConfig);
+/**
+ * For a given package, generate the outputs we want.
+ * - CommonJS
+ * - ES6
+ *
+ * https://redfin.engineering/node-modules-at-war-why-commonjs-and-es-modules-cant-get-along-9617135eeca1
+ * Per the above blog, we could possibly do better for our consumers by making
```

Comment:
I'll probably need to modify this since libraries in here might be for both node and browsers in some cases and so they'll want to specify what their output should be. I'll play with that when we have actual code.

Reply:
I suspect we'll want to read the relevant package.json and look for a field of some kind to determine that.